install.ps1

```diff
@@ -840,6 +841,42 @@ foreach ($env in $configXml.config.envs.env) {
 }
 refreshenv
 
+Write-Host "[+] Checking for custom Start Layout file..."
+$customLayoutPath = Join-Path $desktopPath "CustomStartLayout.xml"
```

Comment:
I suggest to write this file directly to `${Env:VM_COMMON_DIR}` or at least that we copy it there. For `config.xml`, we download it to the desktop first and then we save it to that directory (including additional packages possibly). I think the original reason to do it this way is that it is easier for the user to check if this file has been downloaded correctly (maybe @MalwareMechanic has more info on this). But as we display the path in the warning, I think this should be ok.

Reply:
I added a copy command into my function so that if a custom path/file is provided, it will be copied to `${Env:VM_COMMON_DIR}` or if a URL is provided (or the default is used), it will be downloaded to there as well.